src/utils/Utf8.h

```diff
@@ -20,11 +21,95 @@ namespace drafter
     {
         namespace utf8
         {
+
             using codepoint = std::uint32_t;
 
+            constexpr codepoint replacement = 0xFFFD;
+            constexpr codepoint invalid = 0xFFFFFFFF;
+
+            ///
+            /// Decode utf8 codepoint from octet sequence
+            ///
+            /// @param  b   iterator to begin of octet sequence to be decoded from
```

Comment:
Perhaps it would be nicer to call these variables a more explanatory name? `begin` and `end` vs `b` and `e`?

Reply:
Will rename to `first` and `last`.